src/Http/Controllers/SystemMetricsController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Taecontrol\MoonGuard\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\JsonResponse;
+use Taecontrol\MoonGuard\Models\Site;
+use Taecontrol\MoonGuard\Models\SystemMetric;
+use Taecontrol\MoonGuard\Contracts\MoonGuardSite;
+use Taecontrol\MoonGuard\Repositories\SiteRepository;
+use Taecontrol\MoonGuard\Events\SystemMetricAlertEvent;
+
+class SystemMetricsController extends Controller
+{
+    public function store(Request $request): JsonResponse
+    {
+        /** @var Site */
+        $site = SiteRepository::query()
+            ->where('api_token', $request->input('api_token'))
+            ->first();
+
+        abort_if(! $site, 403);
+
+        SystemMetric::create([
+            'cpu_usage' => $request->input('cpuLoad'),
+            'memory_usage' => $request->input('memory'),
+            'disk_usage' => json_encode($request->input('disk')),
+            'site_id' => $site->id,
+        ]);
+
+        $this->checkLimits($site, $request);
+
+        return response()->json([
+            'success' => true,
+        ]);
+    }
+
+    private function checkLimits(MoonGuardSite $site, Request $request)
```

Comment:
Share the needed properties (be more precise):

```suggestion
    private function checkLimits(MoonGuardSite $site, int cpuLoad, int memory, int diskUsagePercentage)
```